roles/reboot/tasks/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
+# provide an empty iterator when a list is not provided
+# http://docs.ansible.com/ansible/playbooks_conditionals.html#loops-and-conditionals
 - include: ../../../common/ans_reboot.yaml
+  vars:
+    services: "{{ wait_for_services|default([]) }}"
```

Comment:
Shouldn't this be called `wait_for_services` here and in reboot? Otherwise when you use the role you have to use one and when you use the include another, right? Would be nicer to be consistent everywhere.